thousandeyessdk/test.py

```diff
@@ -30,5 +30,24 @@ def domain(self):
     def country_id(self):
         return self._data.get('countryId', None)
 
+    def get_detailed_path_trace(self, agent_id, round_id):
+        """
+        Request
+            Required parameters:
+
+            {agentId} the ID of the endpoint agent from which you wish to obtain data
+            {roundId} the round ID for which you wish to obtain data. Equals the beginning of the testing round, in epoch time format.
+            There is no request body for this request.
+        Response
+            Each route should start with a hop of 1
+            Where a hop number is missing from response data, this is an indication that a star (*) response was returned in the path trace attempt for that hop.
+        """
+        if self._api.version != 6:
```

Comment:
this api is available only in version 7, so this should be != 7

Reply:
both versions provide `/net/path-vis`
https://developer.thousandeyes.com/v7/test_data/#/path-vis
https://developer.thousandeyes.com/v6/test_data/#/traceroute

the `f"/net/path-vis/{self.id}/{agent_id}/{interval}"` suggests that it's v6 🤔